### ISSUE TITLE
[BUG] `getLogger()` is not creating logs in apps.

### ISSUE THREAD

**Issue by PrajvalRaval** [RocketChat/Rocket.Chat.Apps-engine#294]
I implemented `app.getLogger()` in my app but nothing shows up when I check the logs using the `View Logs` button of the app.

I tried using all `app.getLogger().error()`, `app.getLogger().log()`, and `app.getLogger().info()` methods but still nothing shows up. Even checked the MongoDB - `rocketchat_apps_logs` collection and no elements are being created here in `args` array.

Only the following default logs are being printed and not the one's I created.

![Screenshot 2020-08-06 at 7 24 35 PM](https://user-images.githubusercontent.com/41849970/89540317-7f9d8380-d81a-11ea-8a29-2050a6934ea9.png)

Using "@rocket.chat/apps-engine": "^1.16.0" and Rocket Chat v3.5.0.

**Comment by d-gubert** [comment]
Can you provide us with the code that you're trying to execute with the logger? It might help us find the issue you're facing

**Comment by PrajvalRaval** [comment]
Yeah sure @d-gubert, however, I did replace the code that I was using to just use `console.log()` instead of `getLogger()`, since it was not working for me. 

I am using it at multiple places in my app. But one example is here -> https://github.com/PrajvalRaval/Apps.Salesforce.LiveAgent/blob/master/lib/SalesforceAgentAssignedHelpers/SubsribeToLiveAgentHelper.ts#L80

Sorry, but you will have to search other  `console.log` 's from the app yourself.  😄
